README.md

```diff
@@ -83,11 +83,11 @@ The ESM build of the library supports tree shaking thus having no side effects.
 Variant A:
 
 ```js
-import Chart from 'chart.js';
```

Comment:
I not much of fan of the /auto variant, since it removes the idea of tree shaking in chart.js